programmers/REACT-TASK-APP/src/index.css

```
@@ -0,0 +1,3 @@
+:root {
+
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)